### PR TITLE
Releases

### PR DESCRIPTION
**Pull request by apandada1**
Now the PPA is active with the latest source from master branch

### CODE REVIEW

**Comment by LordAmit** [comment]
I actually completed merging changes from the master branch (after making some mistakes :monkey: )

Maybe try pulling my version and see if everything is ok?

**Comment by LordAmit** [comment]
Also, this version includes older copyright and homepage addresses. I think it is better if you pull the releases to yours and see if everything is alright.

**Comment by apandada1** [comment]
I added some commits which now fixes copyright and homepage

File: debian/copyright [hunk 1]
```diff
@@ -3,7 +3,7 @@ Upstream-Name: Brightness Controller
 Upstream-Contact: Archisman Panigrahi <apandada1@gmail.com>
 
 Files: *
-Copyright: 2017, Amit Seal Ami <amitseal@outlook.com>
+Copyright: 2016, Amit Seal Ami <amitseal@outlook.com>
```
Comment (LordAmit) [inline debian/copyright]: This needs to be 
2017, Amit Seal Ami <amitsealami@gmail.com>

File: usr/share/brightness-controller/ui/about.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def retranslateUi(self, Form):
 "<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:16pt;\">Brightness Controller v2.0 build 1</span></p>\n"
 "<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-weight:600;\">Author: </span>Amit Seal Ami, </p>\n"
 "<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Bangladesh, </p>\n"
-"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">amitseal@outlook.com</p>\n"
+"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">amitsealami@gmail.com</p>\n"
```
Comment (apandada1) [inline usr/share/brightness-controller/ui/about.py]: Update this in the master branch as well

**Comment by apandada1** [comment]
Done

**Pull request merged**